### PR TITLE
Install dumb-init from download and update to 1.0.2

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: debian-base.dockerfile [hunk 1]
```diff
@@ -6,9 +6,18 @@ ADD ./common/scripts /scripts
...
-ADD ./common/dumb-init_${DINIT_VERSION}_amd64 /usr/local/bin/dumb-init
-RUN ln -s /usr/local/bin/dumb-init /usr/local/bin/dinit
+ENV DINIT_VERSION "1.0.2"
+RUN set -x \
+    && apt-get-install.sh ca-certificates curl \
+    && cd /tmp \
+    && DINIT_DEB_FILE="dumb-init_${DINIT_VERSION}_$(dpkg --print-architecture).deb" \
+    && curl -sSL -O "https://github.com/Yelp/dumb-init/releases/download/v$DINIT_VERSION/$DINIT_DEB_FILE" \
+    && curl -sSL "https://github.com/Yelp/dumb-init/releases/download/v$DINIT_VERSION/sha256sums" | \
+        grep "\s\*\?$DINIT_DEB_FILE$" | sha256sum -c - \
```
Comment (jerith) [inline debian-base.dockerfile]: Is it worth comparing checksums that come from the same place as the thing we're downloading? If one is compromised, the other will probably be compromised as well.

**Comment by jerith** [comment]
One comment, but :+1: either way.

File: debian-base.dockerfile (hunk 1 above)
Comment (JayH5) [inline debian-base.dockerfile]: Unless the checksum is signed it's not much good in terms of checking authenticity. There may be value in checking integrity, though.

Comment (jerith) [inline debian-base.dockerfile]: We can generally trust TCP to deliver the right bytes. If it doesn't, the package install will probably fail anyway.

**Comment by jerith** [comment]
:+1:

**Pull request merged**